CHANGES.md

```diff
@@ -24,6 +24,13 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.5.0
+
+- Support for causal, counterfactual, and error analysis tools
```

Comment:
According to the guide above this should be grouped by category (see v0.4.0 for an example)

Also, what was the package renamed to? And which package? We ship several out of this repo.

Reply:
Oh I see how that works, updated. It looks like people haven't been updating the v-next section as intended.. I'll go through and try to add as many details as possible from the commit history.

Reply:
Yes we've been very sloppy about this...